core-libs/setup/recipes/b2b/b2b-storefront.module.ts

```diff
@@ -37,7 +35,14 @@ import { organizationLayoutConfig } from './config/organization-layout.config';
     provideDefaultConfig(defaultB2bOccConfig),
     provideDefaultConfigFactory(defaultCmsContentConfig),
     provideDefaultConfig(defaultB2bCheckoutConfig),
+    provideDefaultConfig({
```

Comment:
duplicate, we have this a few lines up as well.

Reply:
thanks, fixed